flambe/nn/cnn.py

```diff
@@ -3,24 +3,24 @@
 from torch import nn
 from torch import Tensor
 
-from flambe.nn.module import Module
+from flambe.nn.module import Encoder
 
 
 def conv_block(conv_mod: nn.Module,
                activation: nn.Module,
-               pooling: nn.Module,
                dropout: float,
+               pooling: Optional[nn.Module] = None,
                batch_norm: Optional[nn.Module] = None) -> nn.Module:
     """Return a convolutional block.
 
     """
 
     mods = [conv_mod]
 
-    if pooling:
+    if pooling is not None:
```

Comment:
Why go this path?

Reply:
pooling is a Module. I generally prefer using `is not None` on non built-in objects, as they may or may not implement the boolean method